### PR TITLE
Travisify

### PR DESCRIPTION
**Pull request by Manishearth**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+language: rust
+sudo: false
+
+
+script:
+ - cargo build
```
Comment (bjz) [inline .travis.yml]: Why do you need to explicitly define the script? I thought that `language: rust` would do that automagically?

**Comment by jdm** [comment]
@Manishearth What's the status here?

**Comment by Manishearth** [comment]
retrying build

**Comment by jdm** [comment]
Yay!

**Pull request merged**